tests/unit/servers/test_confluence_server.py

```diff
@@ -74,6 +73,30 @@ def mock_confluence_fetcher():
     }
     mock_fetcher.add_comment.return_value = mock_comment
 
+    # Mock search_user method
+    mock_user_search_result = MagicMock()
+    mock_user_search_result.to_simplified_dict.return_value = {
+        "entity_type": "user",
+        "title": "First Last",
+        "score": 0.0,
+        "user": {
+            "account_id": "a031248587011jasoidf9832jd8j1",
+            "display_name": "First Last",
+            "email": "first.last@foo.com",
+            "profile_picture": {
+                "path": "/wiki/aa-avatar/a031248587011jasoidf9832jd8j1",
+                "width": 48,
+                "height": 48,
+                "isDefault": False,
+            },
```

Comment:
The mock for `profile_picture` here returns a dictionary object, but the actual `ConfluenceUserSearchResult.to_simplified_dict()` method (via `ConfluenceUser` model) would return a string path for this field.

https://github.com/sooperset/mcp-atlassian/blob/ad481263e78ca2671f4fa3d8f724c4c8b9f3e801/src/mcp_atlassian/models/confluence/common.py#L59-L62

```suggestion
           "profile_picture": "/wiki/aa-avatar/a031248587011jasoidf9832jd8j1",
```